package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
```

Comment:
It uses spread properties for object literals, for the directAttributes. It is actually stage-3 now, so could change that actually.